### PR TITLE
Add JobId to response of sync Job execution

### PR DESCRIPTION
**Pull request by AndroidJVCDER**
It would be nice to have the jobid in the response of sync Job execution. We use a message brooker to start jobs and the jobId would help to identifier.

### CODE REVIEW

**Comment by quanpinjie** [comment]
Great job

**Comment by zeitos** [comment]
Nice, @AndroidJVCDER can you fix the tests so we can merge this?

File: job-server/config/docker.conf [hunk 1]
```diff
@@ -14,7 +14,7 @@ spark {
     port = 8090
     jobdao = spark.jobserver.io.JobSqlDAO
 
-    context-per-jvm = true
+    context-per-jvm = false
```
Comment (zeitos) [inline job-server/config/docker.conf]: I don't think this change goes into this PR.
Please revert

**Comment by zeitos** [comment]
@AndroidJVCDER can you merge with master again to see if tests pass?

**Comment by hntd187** [comment]
@AndroidJVCDER can you rebase off master?

**Comment by f1yegor** [comment]
I think this PR could be closed, because functionality already merged by my PRs.

**Pull request closed without merging**